proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -694,6 +710,15 @@ message AsOfJoinTablesRequest {
   MatchRule as_of_match_rule = 7;
 }
 
+message AjRajTablesRequest {
+  Ticket result_id = 1;
+  TableReference left_id = 2;
+  TableReference right_id = 3;
+  repeated string exact_match_columns = 4;
+  string as_of_column = 5;
```

Comment:
as it seems this is a boolean column expr, either the name or docs should make this clear. i assume this is validated to ensure it is safe for arbitrary clients to pass it in (and that both of the other "column" fields are only names and not exprs)?

Reply:
Added docs.